[2025-02-16T13:23:04.066+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-16T13:23:04.131+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: run_operator_dag.task2 manual__2025-02-16T13:22:43.063555+00:00 [queued]>
[2025-02-16T13:23:04.175+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: run_operator_dag.task2 manual__2025-02-16T13:22:43.063555+00:00 [queued]>
[2025-02-16T13:23:04.175+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-02-16T13:23:04.224+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): task2> on 2025-02-16 13:22:43.063555+00:00
[2025-02-16T13:23:04.232+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=5422) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-16T13:23:04.244+0000] {standard_task_runner.py:72} INFO - Started process 5432 to run task
[2025-02-16T13:23:04.241+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'run_operator_dag', 'task2', 'manual__2025-02-16T13:22:43.063555+00:00', '--job-id', '888', '--raw', '--subdir', 'DAGS_FOLDER/10_run_operator_dag.py', '--cfg-path', '/tmp/tmpv7jinape']
[2025-02-16T13:23:04.245+0000] {standard_task_runner.py:105} INFO - Job 888: Subtask task2
[2025-02-16T13:23:04.364+0000] {task_command.py:467} INFO - Running <TaskInstance: run_operator_dag.task2 manual__2025-02-16T13:22:43.063555+00:00 [running]> on host ab74d4d90d32
[2025-02-16T13:23:04.596+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='run_operator_dag' AIRFLOW_CTX_TASK_ID='task2' AIRFLOW_CTX_EXECUTION_DATE='2025-02-16T13:22:43.063555+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-16T13:22:43.063555+00:00'
[2025-02-16T13:23:04.598+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-16T13:23:04.649+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-02-16T13:23:04.651+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'sleep 5']
[2025-02-16T13:23:04.687+0000] {subprocess.py:99} INFO - Output:
[2025-02-16T13:23:09.727+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-02-16T13:23:09.790+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-16T13:23:09.791+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=run_operator_dag, task_id=task2, run_id=manual__2025-02-16T13:22:43.063555+00:00, execution_date=20250216T132243, start_date=20250216T132304, end_date=20250216T132309
[2025-02-16T13:23:10.037+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-16T13:23:10.092+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-16T13:23:10.106+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
